pages/hydra/developer_resources/what-happens-deposit.md

```diff
@@ -7,3 +7,15 @@ folder: hydra/how-to/
 sidebar: home_sidebar
 tags: [development_resources]
 ---
+
+
+1. I'm on the form to create a new work and I click the submit.
+1. The data is submitted to the controller for the correct work type.  This controller was generated when you did `bin/rails generate hyrax:work <type>`.  For example the `app/controllers/hyrax/generic_works_controller.rb` if you generated a work type of `GenericWork`.
+1. This controller is including `Hyrax::WorksControllerBehavior`, so the `create` method is run: https://github.com/projecthydra-labs/hyrax/blob/master/app/controllers/concerns/hyrax/works_controller_behavior.rb#L57-L69
```

Comment:
Can you make clickable links?  You would think the second one would be automatically, but it isn't.  
```
[Hyrax::WorksControllerBehavior](https://github.com/projecthydra-labs/hyrax/blob/master/app/controllers/concerns/hyrax/works_controller_behavior.rb)
[create](https://github.com/projecthydra-labs/hyrax/blob/master/app/controllers/concerns/hyrax/works_controller_behavior.rb#L57-L69)
```